package.json

```diff
@@ -13,6 +13,7 @@
     "lint": "eslint . && echo \"eslint: no lint errors\"",
     "db:migrate": "knex migrate:latest",
     "db:dump": "docker-compose exec postgres pg_dump -s -U development > db/structure.sql",
+    "db:drop": "babel-node scripts/truncateAll.js",
```

Comment:
I think db:truncate is more appropriate, we will need db:drop and db:create too.